src/tests/sql/init_version_5.sql

```diff
@@ -0,0 +1,10 @@
+DROP TABLE objectprocessorqueue;
+
+CREATE TABLE IF NOT EXISTS `knownnodes` (
```

Comment:
Cool (; I haven't seen this used
```
$ sqlite3 .config/PyBitmessage/messages.dat
SQLite version 3.34.1 2021-01-20 14:10:07
Enter ".help" for usage hints.
sqlite> select * from knownnodes;
Error: no such table: knownnodes
```

Reply:
I think the table was deleted in a later version.

Reply:
It looks like some weird leftover of deleted code. Since the current upgrade code deletes it, we should keep it, otherwise we'd have to test against ancient versions.